### DISCUSSION TITLE
Monthly ESMValTool community meeting - November 2024

### DISCUSSION THREAD

**Discussion by paultgriffiths** [ESMValGroup/Community#191] [Meetings]
After a longer-than-expected hiatus, for which apologies, the next ESMValTool Community meeting will be in November.

The meeting will be at 1200 GMT on 2024-11-25 (Monday).

The duration will be one hour.

Please join the meeting at 
https://uni-bremen.zoom-x.de/j/66061801859?pwd=s3r3axGbkToaoZXcfAaGGQpFeXbbtl.1
Meeting ID: 660 6180 1859
Passcode: 46286

## Format and agenda

Anyone with an interest in ESMValTool is welcome to join and ask questions

We start the meeting by doing a round where everyone gets one minute to introduce themselves and tell what they will be working on in the next month or what they would like help with.

Using the input from the introduction round, we choose some topics to talk about in more detail.

If you already know that you would like to bring up a particular topic in the meeting, you can mention the GitHub issue on the topic in a comment below. That will make it easy to find for the other people in the meeting and we can already start discussing there before the meeting. Just create a new GitHub issue about your topic if there is no open issue yet.

GitHub issues are our preferred way to communicate outside the meeting and keep track of work, so any feature requests/action items/etc discussed in the meeting should ideally result in a GitHub issue.

Notes from the previous meeting are available in https://github.com/ESMValGroup/Community/discussions/161

### Discussion on ESMValTool governance:

Our current governance document is at :https://esmvaltool.org/team/

Some ideas for re-drafting our document is at: https://sustainers.github.io/governance-readiness/
It would be good to have a discussion on how to write our governance structure. Bouwe is best placed to lead this topic, so please tag him.

### Discussion on preparations for ESGF phase out of CMIP5 and CMIP3 data:

This is described in a discussion and it will be good for the community to provide input here: https://github.com/ESMValGroup/Community/discussions/181

### UKRI ending NCAS' Zoom subscription

@paultgriffiths will be unable to set up Zoom meetings using my NCAS account in the future.  Anyone else with a Pro subscription care to set up the meetings? I think this means I will also step down as convener of the monthly Community meetings after this meeting.

**Comment by paultgriffiths** [comment]
I regret I can't make the meeting as planned - can someone else please chair?

**Reply by bettina-gier** [reply]
I can chair it, can you add me (gier@uni-bremen.de) as a co-host to your zoom link, so we don't need to change it on such short notice?

Edit: Just realized that apparently you cannot appoint a co-host from a different licenced organization. Is your zoom meeting set up to start without host or should I make another one?

**Reply by paultgriffiths** [reply]
Yes please make a new one and if you can, please post the meeting link here.  I didn't send out ical meeting invitations this time, so the link on this page should be sufficient.

**Reply by bettina-gier** [reply]
Maybe I was too late, but nobody joined for 20min, so I'll call it off here!

**Reply by paultgriffiths** [reply]
Sorry to hear this.
